deploy/examples/object.yaml

```
@@ -116,6 +116,7 @@ spec:
       disabled: false
     readinessProbe:
       disabled: false
+  # RGWOpsLogEnabled: true
```

Configs should not start with capital letters by k8s manifest standards.

```suggestion
  # rgwOpsLogEnabled: true
```